docs/api.md

```diff
@@ -1,468 +1,427 @@
 
-##### Example Response
-```json
-HTTP/1.1 202 OK
-Content-Type: application/json
+No query parameters are supported.
 
-{
-  "diff": []
-}
-```
+### [Jobs](https://v3-apidocs.cloudfoundry.org/version/3.117.0/#jobs)
 
-### Buildpacks
-| Resource | Endpoint |
-|--|--|
-| List Buildpacks | GET /v3/buildpacks |
+#### [Get a job](https://v3-apidocs.cloudfoundry.org/version/3.117.0/#get-a-job)
 
-#### [List Buildpacks](https://v3-apidocs.cloudfoundry.org/version/3.117.0/index.html#list-buildpacks)
-```bash
-curl "http://localhost:9000/v3/buildpacks?order_by=postion" \
-  -X GET
-```
+> **Warning**
+> This endpoint always returns an empty resource with `state: "COMPLETE"`.
 
-### Log-Cache API
-We support basic, unauthenticated versions of the following [log-cache](https://github.com/cloudfoundry/log-cache) APIs that return hard-coded responses.
-
-#### [Log-Cache Info](https://github.com/cloudfoundry/log-cache#get-apiv1info)
-
-| Resource                     | Endpoint         |
-| ---------------------------- | ---------------- |
-| Retrieve version information | GET /api/v1/info |
-
-#### [Log-Cache Read](https://github.com/cloudfoundry/log-cache#get-apiv1readsource-id)
-
-| Resource                                                                                        | Endpoint                     |
-| ----------------------------------------------------------------------------------------------- | ---------------------------- |
-| Retrieve data by source-id. Currently returns a hard-coded empty list of Loggregator envelopes. | GET /api/v1/read/<source-id> |
-
-
-### Service Instances
-
-Docs: https://v3-apidocs.cloudfoundry.org/version/3.117.0/index.html#service-instances
-
-| Resource                     | Endpoint         |
-| ---------------------------- | ---------------- |
-| Create Service Instance | POST /v3/service_instances |
-| List Service Instance | GET /v3/service_instances |
-
-#### [Create Service Instances](https://v3-apidocs.cloudfoundry.org/version/3.117.0/index.html#create-a-service-instance)
-Currently, we support creation of user-provided service instances
-```bash
-curl "https://localhost:9000/v3/service_instances" \
-  -X POST \
-  -H "Authorization: bearer [token]" \
-  -H "Content-type: application/json" \
-  -d '{
-    "type": "user-provided",
-    "name": "my_service_instance",
-    "credentials": {
-      "foo": "bar",
-      "baz": "qux"
-    },
-    "tags": ["foo", "bar", "baz"],
-    "syslog_drain_url": "",
-    "route_service_url": "",
-    "metadata": {
-      "annotations": {
-        "foo": "bar"
-      },
-      "labels": {
-        "baz": "qux"
-      }
-    },
-    "relationships": {
-      "space": {
-        "data": {
-          "guid": "7304bc3c-7010-11ea-8840-48bf6bec2d78"
-        }
-      }
-    }
-  }'
-```
+### [Manifests](https://v3-apidocs.cloudfoundry.org/version/3.117.0/#manifests)
+
+#### [Applying a manifest](https://v3-apidocs.cloudfoundry.org/version/3.117.0/#create-a-route)
```

Comment:
the link is incorrect. It need to point to - https://v3-apidocs.cloudfoundry.org/version/3.117.0/#apply-a-manifest-to-a-space

Reply:
Fixed, thanks!